assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -798,8 +798,10 @@ def __init__(
                 if connection.type != "azure_open_ai":
                     raise Exception(f"Received unexpected endpoint type {connection.type}"
                                     "only Azure Open AI endpoints are supported at this time")
-
-                self.api_version = connection.metadata["ApiVersion"]
+                api_version = "2023-07-01"
```

Comment:
did you test this value, I remember it has preview in the string, could you please double confirm?

Reply:
yes preview works : https://ml.azure.com/experiments/id/82fe6b93-2b2d-489c-917a-832ee08abef4/runs/busy_basil_d1d284nfvp?wsid=/subscriptions/15ae9cb6-95c1-483d-a0e3-b1a1a3b06324/resourceGroups/copilot-demo-eastus/providers/Microsoft.MachineLearningServices/workspaces/aistudio_townhall&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#